editor/components/rich-text/index.js

```diff
@@ -122,6 +123,8 @@ export class RichText extends Component {
 		this.onPaste = this.onPaste.bind( this );
 		this.onCreateUndoLevel = this.onCreateUndoLevel.bind( this );
 		this.setFocusedElement = this.setFocusedElement.bind( this );
+		this.getFocusPosition = this.getFocusPosition.bind( this );
```

Comment:
Remnant? Cannot see this in this diff.

Reply:
Thanks for catching that. Removed in https://github.com/WordPress/gutenberg/pull/5794/commits/f4a4358dbfeba0e5a1770a1632a5c5d920ccca91.